pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -235,6 +238,8 @@ type EnvoyConfig struct {
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 	// Envoy admin port
 	AdminPort *int32 `json:"adminPort,omitempty"`
+	// PodDisruptionBudget attached to Envoy Deployment(s)
+	DisruptionBudget DisruptionBudget `json:"disruptionBudget,omitempty"`
```

Comment:
Since the PDB works a bit differently in case of kafka broker pods I'd not use the `DisruptionBudget` struct for both kafka broker pod and envoy pods. I'd rather introduce a new type that derives from `DisruptionBudget` struct and introduce the `Strategy` field in the derived struct.

Reply:
Done, I created a secondary Structure called `DisruptionBudgetWithStrategy` that is a composition of the initial `DisruptionBudget` and the field strategy